cmd/isolateDAG.go

```diff
@@ -144,6 +144,7 @@ func isolateGraph(graph *fs.Graph) {
 		"debug",
 		"docs",
 		"dbt_modules",
+		"macros/sampling.sql",
```

Comment:
Would it also work if the whole `"macros"` folder is in `projectFiles`? The macros folder is quite likely to exist for any dbt set-up & unused macros will just be ignored (rather than specifying a very specific `.sql` file)?

Reply:
Good suggestion, I've added the whole directory instead and made sure we don't try and symlink macros.